sdk/core/Azure.Core/src/Pipeline/BearerTokenChallengeAuthenticationPolicy.cs

```diff
@@ -33,16 +42,17 @@ internal class BearerTokenChallengeAuthenticationPolicy : HttpPipelinePolicy
         /// Creates a new instance of <see cref="BearerTokenChallengeAuthenticationPolicy"/> using provided token credential and scopes to authenticate for.
         /// </summary>
         /// <param name="credential">The token credential to use for authentication.</param>
-        /// <param name="scopes">Scopes to authenticate for.</param>
+        /// <param name="scopes">_scopes to authenticate for.</param>
```

Comment:
nit: I don't think you meant to change the casing here. Also, perhaps we could word it in a more natural way, somthing like, 

> Scopes to be included in acquired tokens.

What do yo think?

Reply:
I ended leaving just the private _scopes and removing the comment.